clarity/src/vm/docs/mod.rs

```diff
@@ -2192,6 +2210,7 @@ fn make_for_special(api: &SpecialAPI, name: String) -> FunctionAPI {
         signature: api.signature.to_string(),
         description: api.description.to_string(),
         example: api.example.to_string(),
+        version: ClarityVersion::Clarity1,
```

Comment:
This isn't accurate for all special functions -- some of them are `Clarity2`.

I'd suggest something like:

```rust
fn make_for_special(api: &SpecialAPI, function: &NativeFunctions) -> FunctionAPI {
    FunctionAPI {
        name: function.get_name(),
        input_type: api.input_type.to_string(),
        output_type: api.output_type.to_string(),
        signature: api.signature.to_string(),
        description: api.description.to_string(),
        example: api.example.to_string(),
        version: function.get_version(),
```

Reply:
did this